CORE/world_extensions/core_world.rb

```diff
@@ -60,10 +59,10 @@ def create_new_browser
     def create_chrome_browser
         caps = Selenium::WebDriver::Remote::Capabilities.chrome(chrome_options: {'detach' => true})
 
-        if @configuration['USE_GRID'] == 'true'
-            driver = Selenium::WebDriver.for(:remote, :url => environment['GRID'] , desired_capabilities: caps)
+        if @configuration['USE_GRID']
```

Comment:
Once that is pulled out, and all of these if statements are next to each other, it should be way easier to refactor